### DISCUSSION TITLE
Crowdfunding the pest.rs domain renewal

### DISCUSSION THREAD

**Discussion by tomtau** [pest-parser/pest#717] [Announcements]
[The pest.rs domain has expired](https://github.com/pest-parser/pest/issues/714). For the moment, I have updated most of the links to point to [pest-parser.github.io](https://pest-parser.github.io) -- if you cannot access the website (i.e. it still redirects to pest.rs), you can try clearing out your browser cache.

I have also contacted @dragostis via email and he mentioned he was previously covering the domain registration costs (~USD100 per year) out of his own pocket. That doesn't seem like a solution that will work in the long-term, so his suggestion was to crowdfund the domain renewal cost.

**You can help to contribute to this effort either on [GitHub Sponsors](https://github.com/sponsors/pest-parser) or on [Open Collective](https://opencollective.com/pest)**. Thanks!

**Comment by netshade** [comment]
Cost should be covered now - one of the maintainers should likely re-register it?

**Reply by tomtau** [reply]
That's right, the cost should be covered for at least a year now! 🙌 big thanks to all sponsors! 🙏 

I emailed @dragostis and he mentioned he'd try to renew the domain soon, probably today or tomorrow.

**Reply by tomtau** [reply]
It should be back up now: https://pest.rs

**Reply by netshade** [reply]
Awesome, thx
